src/assets/global.scss

```diff
@@ -1,17 +1,24 @@
+@import "assets/colors.scss";
+
 .page-content-container {
-    border: 1px solid #dfe1e1;
-    border-radius: 0.375rem;
-    padding: 20px 0;
-    box-shadow: 0px 3px 12px 0px rgba(0, 0, 0, 0.16);
+  border: 1px solid #dfe1e1;
```

Comment:
Define this color as a variable. @AuraAlba

Reply:
done